### ISSUE TITLE
Fewer Pokemon are Showing up on Map

### ISSUE THREAD

**Issue by Natglobe** [AHAAAAAAA/PokemonGo-Map#3078]
I've seen a lot less Pokemon on the map than I usually do. Less than a week ago it seemed everything was showing up. I've filtered out all the common ones so now I may see a Dratini or an Abra every 5 minutes. I have no error messages, and everything is running smoothly in the terminal.
## Expected Behavior

Map would show all Pokemon within scan radius.

## Current Behavior

Map shows much fewer Pokemon, more useful to go out and hope to find Pokemon yourself.

## Your Environment

* Environment name and version: Python 2.7
* Operating System and version: Mac OS X

Now I do not know if this would help but this is the command I type in: 
python runserver.py -a ptc -u "Username" -p "Password" -l "Location" -t 3 -st 2 -sd 2

**Comment by Scalpweb** [comment]
You should try with "-t1 -sd 5" instead of "-t 3 -sd 2"

**Comment by Matsu16** [comment]
I  am running multiple accounts, but it stil seems like the common are there but the rare.....maby 1..

**Comment by Natglobe** [comment]
@Scalpweb I am scanning now, I'll let you know if I see anything different.

**Comment by Natglobe** [comment]
@Matsu16 Does running multiple accounts help within the same search radius? Or does it just expand it?

**Comment by rlyons20** [comment]
i agree.  I'm seeing a LOT less.  I was considering upgrading my code to the latest, but now I'm not, based on what you're saying...

**Comment by Natglobe** [comment]
@Scalpweb I've noticed a significant change in the amount that are showing up. Thank you!

**Comment by Scalpweb** [comment]
"-sd 5" is really important, because scan delay has been changed to 5s on Niantic server. You should not try to use more than 1 thread with a single account for the same reason : if you try to scan more than 1 cell every 5 seconds, you'll get empty results. If you want to use more than 1 thread, you'll need more than 1 account.  See #2956

**Comment by Matsu16** [comment]
@Natglobe mulitple accounts makes it a lot faster. so 1 account is 1 step. so 10 accounts = 10 steps. with -sd 5 offcourse.  the more accounts the bigger ur search area.

**Comment by rlyons20** [comment]
how do you use multiple accounts??

**Comment by jesusdrodriguez** [comment]
-u firstaccount -u secondaccount -u thirdblah -u blah -ublah2 -p passwordforall

**Comment by jfla15** [comment]
@Matsu16 Steps aren't really bound to accounts. You can have as many steps as you want with one account, but it will take really long to complete one spiral. The aforementioned PR #2956 means that its actually threads that equal the amount of accounts. That's why it speeds up with more.

**Comment by Matsu16** [comment]
run the latest dev. with -a ptc/google -u USERNAME -p PASS -a ptc/google -u USERNAME -p PASS  etc.. latest dev removes the -t function . 1 account = -t 1 and 10 accounts= -t 10, its build in.

**Comment by rlyons20** [comment]
Wow!  Ok!  I will try that!

**Comment by Matsu16** [comment]
https://github.com/sebastienvercammen/nintendo-ptc-account-generator/ or just make some.

**Comment by Exxile22** [comment]
So it's 1 thread per account now? Not 3?

**Comment by jesusdrodriguez** [comment]
@Exxile22 yeah

**Comment by Matsu16** [comment]
latest dev.  yes

**Comment by bambusbo** [comment]
When i try this, it signs in without problems on all accounts, but i cannot see a map in the web map, just a big white box, no map

**Comment by Natglobe** [comment]
@bambusbo The threads that are somewhat related to your issue that I know of are: #2824 #2721 & #3019

**Comment by rlyons20** [comment]
so, mine only signs in the last account in the list.

i tried all these, with no success:

-a google -u username -u username... -p password
-a google -u username -a google -u username... -p password
-a google -u username -p password -a google -u username -p password...

**Comment by bambusbo** [comment]
@rlyons20 remember to use dev version

**Comment by rlyons20** [comment]
oh SHIT.  thanks, bro.

**Comment by MonsterMMORPG** [comment]
@Matsu16  when multiple account is provided, does it scan same location with all accounts or how does it handle?

**Comment by Matsu16** [comment]
I think every account is one green cirkel. So 10 accounts 10 cirkels up.

**Comment by jfla15** [comment]
@MonsterMMORPG @Matsu16 As mentioned before the number of circles aren't bound to the number of accounts. But the more accounts you have active the more threads will be running to complete your scan.
What basically happens is that each location that needs to be scanned sits in a queue and waits for a worker thread to be ready to scan it.

**Comment by MonsterMMORPG** [comment]
@jfla15 so what command do i need to start multiple accounts with single console? i dont want to manually start multiple accounts with each one getting a different lat long

**Comment by jfla15** [comment]
@MonsterMMORPG Try the latest development branch and then the format for multiple accounts is simple adding more -a -u -p. You can use the same authentication method and password for all accounts, or you can mismatch it however you want.
For example:
` python runserver.py -a google -u foobar@gmail.com -p foobarpass -a google -u foobar2@gmail.com -p barfoopass1 -a ptc -u pokefoobar -p pokepass -st 12 -k xyz -l "Aarhus, 8000, Denmark" -sd 5`

**Comment by MonsterMMORPG** [comment]
@jfla15  i have tried but it gives error Please run "grunt build" before starting server

**Comment by jfla15** [comment]
@MonsterMMORPG Oh yeah you're right, grunt is now required on the dev branch.
You can [follow the wiki](https://github.com/AHAAAAAAA/PokemonGo-Map/wiki) for instructions on installing grunt and dependencies.

**Comment by Scalpweb** [comment]
You can also enter your account credentials inside the config.ini file.
If you want to use a lot of accounts, it will be easier.

**Comment by wardude202** [comment]
How do i know if has logged on with all my accounts?

**Comment by LeoPerard** [comment]
@Scalpweb I did it but it is not working (probably did it wrong). The last one is only use. Should the file look like:

```
# Google
auth-service=google
username=google-account@gmail.com
password=google-password
# Pokemon Trainer Club
auth-service=ptc
username=ptc-account
password=ptc-password
```

or

```
auth-service=google ptc
username=google-account@gmail.com ptc-account
password=google-password ptc-password
```
or something else?

**Comment by rlyons20** [comment]
Use the dev branch.  Same problem I was having.  There really should be a sticky somewhere about this...

**Comment by LeoPerard** [comment]
I am using the dev branch but my issue is with the config.ini file. I can set multiple accounts using the command line but I want to set them in the config file :)

**Comment by prinzpi** [comment]
Is it possible to use the same ~20 PTC accounts for 6 or more instances or is this not recommended?
Would it be better to use ~5 PTC accounts for each instance (more effort)?

**Comment by jfla15** [comment]
@wardude202 I don't think there is any efficient way to check if all logins were successful with many accounts right now.
@prinzpi You don't want to use the same accounts for several instances of the server, so go with the latter.
